package.json

```diff
@@ -28,6 +28,8 @@
   },
   "main": "lib/index.js",
   "files": [
+    "dist/",
```

Comment:
I'm using vttjs through npm which will exclude the `dist` folder if it isn't included here.